Library/Homebrew/dev-cmd/boneyard-formula-pr.rb

```diff
@@ -0,0 +1,104 @@
+# Creates a pull request to boneyard a formula.
+#
+# Usage: brew boneyard-formula-pr [options...] <formula-name>
+#
+# Options:
+#   --dry-run:  Print what would be done rather than doing it.
+
+require "formula"
+require "utils/json"
+require "fileutils"
+
+module Homebrew
+  def boneyard_formula_pr
+    formula = ARGV.formulae.first
+    odie "No formula found!" unless formula
+
+    formula_path = formula.path
+    core = formula_path.to_s.include? "homebrew-core"
+    if ARGV.dry_run?
+      ohai "brew update"
+      ohai "brew tap homebrew/boneyard"
+      ohai "cp #{formula_path} #{HOMEBREW_LIBRARY}/Taps/homebrew/homebrew-boneyard"
+      ohai "git rm #{formula_path}"
+      ohai "Adding #{formula.name} to tab migrations" if core
+    else
+      safe_system "brew", "update"
+      safe_system "brew", "tap", "homebrew/boneyard"
+      cp formula.path, HOMEBREW_LIBRARY/"Taps/homebrew/homebrew-boneyard"
+      formula.path.parent.cd { safe_system "git", "rm", formula.path }
+      if core
+        tap_migrations = Utils::JSON.load(File.read(HOMEBREW_LIBRARY/"Taps/homebrew/homebrew-core/tap_migrations.json"))
```

Comment:
`HOMEBREW_LIBRARY/"Taps/homebrew/homebrew-core/tap_migrations.json"` is used several times. Might make sense to put it into a constant (`TAP_MIGRATIONS_PATH`?) at the beginning of the command to make the rest of the code more readable.